Dockerfile

```diff
@@ -35,8 +35,6 @@ RUN apt-get update && \
     wget -O - https://overviewer.org/debian/overviewer.gpg.asc | apt-key add - && \
     apt-get update && \
     apt-get install -y --no-install-recommends minecraft-overviewer && \
-    apt-get clean && \
```

Comment:
Are you sure? I still see (albeit slightly different) it [in there](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run).

> Below is a well-formed RUN instruction that demonstrates all the apt-get recommendations.
> ```
> RUN apt-get update && apt-get install -y \
>    aufs-tools \
>    automake \
>    build-essential \
>    curl \
>    dpkg-sig \
>    libcap-dev \
>    libsqlite3-dev \
>    mercurial \
>    reprepro \
>    ruby1.9.1 \
>    ruby1.9.1-dev \
>    s3cmd=1.1.* \
> && rm -rf /var/lib/apt/lists/*
```

Reply:
I think the `apt-get clean` does refer to running `rm -rf /var/lib/apt/lists/*` because the test results from running dive still don't show any lingering cache files.

Reply:
Oh interesting. I'll look into.